apps/application-system/api/infra/application-system-api.ts

```diff
@@ -146,7 +146,7 @@ export const serviceSetup = (services: {
         '/k8s/application-system-api/PAYMENT_BASE_CALLBACK_URL',
       PAYMENT_ADDITION_CALLBACK_URL:
         '/k8s/application-system-api/PAYMENT_ADDITION_CALLBACK_URL',
-      ARK_BASE_URL: '/k8s/application-system-api/ARK_ENDPOINT',
```

Comment:
So is this parameter not used anymore? Can we delete it?

Reply:
yes, we probably can. i didn't know if something would break, so i just started by leaving a comment about removing it after today. i'll delete it after this goes through

Reply:
you don't have permissions to delete it, I just created https://github.com/island-is/infrastructure/issues/802